### ISSUE TITLE
3.0.0: Error when retrieving info of a video with several dub tracks via Android client.

### ISSUE THREAD

**Issue by wukko** [LuanRT/YouTube.js#322]
### Steps to reproduce

```js
import { Innertube } from 'youtubei.js';

const yt = await Innertube.create();
let info = await yt.getBasicInfo('TJ2ifmkGGus', 'ANDROID');
```

[Video in code sample](https://www.youtube.com/watch?v=TJ2ifmkGGus) has several audio tracks. Any such video causes this error.

### Failure Logs

```shell
node_modules/youtubei.js/dist/src/parser/youtube/VideoInfo.js:65
            throw new InnertubeError('This video is unavailable', info.playability_status);
                  ^

InnertubeError: This video is unavailable
    at new VideoInfo (node_modules/youtubei.js/dist/src/parser/youtube/VideoInfo.js:65:19)
    at Innertube.<anonymous> (node_modules/youtubei.js/dist/src/Innertube.js:73:20)
    at Generator.next (<anonymous>)
    at fulfilled (node_modules/youtubei.js/dist/src/Innertube.js:4:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
{
  info: {
    status: 'ERROR',
    reason: 'This video is unavailable',
    embeddable: false,
    audio_only_playablility: null,
    error_screen: PlayerErrorMessage {
      type: 'PlayerErrorMessage',
      subreason: Text {
        runs: [
          TextRun {
            text: 'Watch on the latest version of YouTube.',
            bold: false,
            italics: false,
            strikethrough: false,
            endpoint: [NavigationEndpoint]
          }
        ],
        text: 'Watch on the latest version of YouTube.'
      },
      reason: Text {
        runs: [
          TextRun {
            text: 'The following content is not available on this app.',
            bold: false,
            italics: false,
            strikethrough: false,
            endpoint: undefined
          }
        ],
        text: 'The following content is not available on this app.'
      },
      proceed_button: null,
      thumbnails: [
        Thumbnail {
          url: '//s.ytimg.com/yts/img/meh7-vflGevej7.png',
          width: 140,
          height: 100
        },
        Thumbnail {
          url: '//s.ytimg.com/yts/img/meh7-vflGevej7.png',
          width: 140,
          height: 100
        }
      ],
      icon_type: 'ERROR_OUTLINE'
    }
  },
  date: 2023-02-25T11:09:05.107Z,
  version: '3.0.0'
}
```

### Expected behavior

I expected to get video info along with all possible dubs, just like what `WEB` client returns.

### Current behavior

InnerTube throws an error and doesn't provide **any** video info if the video in question has several dub tracks.

### Version

Default

### Anything else?

Updating the Android app version [here](https://github.com/LuanRT/YouTube.js/blob/a0bfe164279ec27b0c49c6b0c32222c1a92df5c3/src/utils/Constants.ts#L50) would probably fix this issue, as error suggests.

### Checklist

- [x] I am running the latest version.
- [X] I checked the documentation and found no answer.
- [X] I have searched the existing issues and made sure this is not a duplicate.
- [X] I have provided sufficient information.

**Comment by wukko** [comment]
uhh it works now?? i guess this was temporary, but i couldn't get it to work a few hours ago. that's weird

**Comment by wukko** [comment]
okay, this just happened again. same error, but it doesn't happen 100% of the time.

**Comment by wukko** [comment]
truly no idea what this is caused by, so this specific issue is irrelevant

**Comment by LuanRT** [comment]
See https://github.com/iv-org/invidious/pull/3290.

The library sends all parameters required by the Android client, except the specific user agent, which is likely what's causing the issue here. 

Reopening this until #318 is live. Thanks for reporting!

**Comment by LuanRT** [comment]
Just released v3.1.0, but I'm not sure if this is fixed yet (can't be reproduced in my region). So please do test it if you can.
